DetectorDescription/DDCMS/interface/DDFilteredView.h

```diff
@@ -236,6 +249,9 @@ namespace cms {
     //  speeding up avoiding the same search over
     //  the registry
     const DDSpecPar* find(const std::string&) const;
+    void filter(DDSpecParRefs&, const std::string&) const;
+    const std::string_view front(const std::string_view) const;
+    const std::string_view back(const std::string_view) const;
```

Comment:
Minor, but
* `string_view` by definition provides a `const` access to the underlying "string", so additional `const` does not really add anything (but does no harm either)
* `const` on return values has little value, because the caller can "capure" it to a non-`const` variable with zero cost

Reply:
thanks!